src/util/schemas/ThreadCreateSchema.ts

```diff
@@ -0,0 +1,9 @@
+import { ChannelType } from "../entities";
+
+export interface ThreadCreateSchema {
+	auto_archive_duration?: 60 | 1440 | 4320 | 10080;
```

Comment:
It should be `number` per https://github.com/fosscord/fosscord-server/pull/876#discussion_r958639774